demos/text_detection_demo/cpp/README.md

```diff
@@ -9,6 +9,8 @@ The demo shows an example of using neural networks to detect and recognize print
 * `horizontal-text-detection-0001`, which is a detection network that works much faster than models above, but it is applicable to finding more or less horizontal text only.
 * `text-recognition-0012`, which is a recognition network for recognizing text.
 * `text-recognition-0013`, which is a recognition network for recognizing text. You should add option `-tr_pt_first` and specify output layer name via `-tr_o_blb_nm` option for this model (see model [description](../../../models/intel/text-recognition-0013/README.md) for details).
+* `text-recognition-0014`, which is a recognition network for recognizing text. You should add option `-tr_pt_first` and specify output layer name via `-tr_o_blb_nm` option for this model (see model [description](../../../models/intel/text-recognition-0014/README.md) for details).
+* `text-recognition-0015`, which is a recognition network for recognizing text. You should add option `-m_tr_ss "  #?0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"` (supported symbols set), `-tr_o_blb_nm  "logits"` (to specify output name) `-tr_composite` (model consists of encoder and decoder part) and `-dt simple` (to specify decoder type). You can also specify `-lower` option to convert predicted text to lower-case. See model [description](../../../models/intel/text-recognition-0013/README.md) for details.
```

Comment:
Two spaces are instead of `START_TOKEN` and `PAD_TOKEN`, respectively. These tokens should be ignored during prediction, and model should not predict these classes (at least, before `END_TOKEN`) but to be confident that these tokens would not appear in the prediction I suppress them by using spaces.
You mean, add new parameter `alphabet` and use it as `alphabet` variable in case of composite model (as it is done now with `m_tr_ss`?)

Reply:
I think the explanation with spaces is worth being put in readme because `m_tr_ss` is treated differently for different models. OR in the part where you add `kPadSymbol` to `FLAGS_m_tr_ss` add 3 of `kPadSymbol` for composite model so `SimpleDecoder()` would suppress any of them, explaining in comments that these are 3 symbols actually for.

Should `EncoderDecoderCNN::Init()` wait for `END_TOKEN` instead of fixed `MAX_NUM_DECODER` number of iterations?

> You mean, add new parameter `alphabet` and use it as `alphabet` variable in case of composite model (as it is done now with `m_tr_ss`?)

No, I was saying `alphabet` as flag is more descriptive then `m_tr_ss`. But I didn't notice it was already there, so you can leave it as is.

Reply:
> OR in the part where you add kPadSymbol to FLAGS_m_tr_ss add 3 of kPadSymbol for composite model so SimpleDecoder() would suppress any of them, explaining in comments that these are 3 symbols actually for.  

Done

Reply:
> Should EncoderDecoderCNN::Init() wait for END_TOKEN instead of fixed MAX_NUM_DECODER number of iterations?

Good proposal, I will try to implement it

Reply:
I looked again at the function `EncoderDecoderCNN::Infer`:
Here we have to create result tensor targets and we have to set it's shape in advance, so `END_TOKEN` would only slightly reduce number of decoder runs. This would not significantly boost the performance since the decoder is very lightweight.

Reply:
> OR in the part where you add kPadSymbol to FLAGS_m_tr_ss add 3 of kPadSymbol for composite model so SimpleDecoder() would suppress any of them, explaining in comments that these are 3 symbols actually for.

I still think it's good to explain in code what are the 3 symbols for.

> Here we have to create result tensor targets and we have to set it's shape in advance

You can break before `num_decoder` reaches `MAX_NUM_DECODER` and fill the rest of `targets` with special values.

Reply:
Done. I did not fill `targets` with some meaningful value since the decoder would stop its job after the first entry of `end_token`.